libs/dfi/include/dyn_interface.h

```diff
@@ -50,9 +46,8 @@ typedef struct _dyn_interface_type dyn_interface_type;
 TAILQ_HEAD(methods_head, method_entry);
 struct method_entry {
     int index;
-    char *id;
-    char *name;
-    dyn_function_type *dynFunc;
+    char* id;
```

Comment:
> Is the 'id' generally the name or signature of a method?

It is signature.

> I think it would be good to document (doxygen) this more explicit.

I agree. Currently it is a little confusing. Once automatic descriptor generation is in place, signature should be generated from name.